nuxeo-features/preview/nuxeo-preview-core/src/main/resources/OSGI-INF/properties-contrib.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.preview.properties">
+
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+    <documentation>
+      Property that enables the old JSF preview that allows annotations to be set.
+    </documentation>
+    <property name="nuxeo.old.jsf.preview">false</property>
```

Comment:
Could you also add the documentation and default for `nuxeo.text.annotations` in this config file?